### DISCUSSION TITLE
Nicehash pool reset  1h

### DISCUSSION THREAD

**Discussion by kayansel** [Lolliedieb/lolMiner-releases#1453] [Q&A]
Hello, i need some help!
I am mining on nicehash pool and every hour i have a new dag génération so it like i reset the miner... 
In this case the miner need to re calibrate the card 
i dont know if it possible to fix this issue 
Using HiveOS and lolminer 14.7/14.8
stable OC and same issue on 4 different rig. Tested all nvidia drivers. 
Thanks for help :) 
![lolminer](https://user-images.githubusercontent.com/6828594/161336817-d83a88a4-2f8a-4bb5-92f3-6fb3780727b0.png)

**Comment by BIGguiX** [comment]
it is 1.47's fault.  Go back to 1.46a for now until 1.48 is available in hiveos to test.  It does the same thing for me on ezil.

**Reply by kayansel** [reply]
I have already tested 14.8 on hiveos (manual install) 
![FBF96CCA-3D2E-46A9-9612-475A994FF72E](https://user-images.githubusercontent.com/6828594/161387071-504fc52f-ad9a-4d6f-8237-720bb9b16ea2.jpeg)

**Comment by BIGguiX** [comment]
I was hoping 1.48 would fix this 😢

Le sam. 2 avr. 2022, 10 h 11 a.m., kayansel ***@***.***> a
écrit :
